### PR TITLE
Update Background Gradient styling refguide

### PR DESCRIPTION
**Pull request by UrazAkgultan**

### CODE REVIEW

**Review by ConnorLand — APPROVED** [review]

**Review by ConnorLand — CHANGES_REQUESTED** [review]
Uraz will add a style property before we merge this

**Review by Ahmad-Elsayed — APPROVED** [review]

File: content/en/docs/refguide/mobile/designing-mobile-user-interfaces/native-styling-element-guide.md [hunk 1]
```diff
@@ -1656,22 +1656,41 @@ The default class to style all check box inputs is named `com_mendix_widget_nati
...
+	]
+</colorList>
+<container/>
+<opacity/>
 ```
 
 | Element | Style Properties    | Description |
 | --- | --- | --- |
-| `container` | This has all ViewStyle properties. |        |
+| `angle` | | Line of direction. A value of 0 is equivalent to top; increasing values rotate the design clockwise. |
```
Comment (pratikgoyal9) [inline content/en/docs/refguide/mobile/designing-mobile-user-interfaces/native-styling-element-guide.md]: Would be nice to mention the values range as well. Takes a value from `0` to `360`

**Review by pratikgoyal9 — CHANGES_REQUESTED** [review]

File: content/en/docs/refguide/mobile/designing-mobile-user-interfaces/native-styling-element-guide.md (hunk 1 above)
Comment (pratikgoyal9) [reply content/en/docs/refguide/mobile/designing-mobile-user-interfaces/native-styling-element-guide.md]: Verified and can be resolved.

**Comment by pratikgoyal9** [comment]
Hi @ConnorLand This PR has been reviewed technically from Native-Content side. Once, you are ready to publish to master, you can do so. We would release the updated widget today.

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**